vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -493,7 +493,9 @@ bool FrameworkElementViewManager::UpdateProperty(
             states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Expanded)] =
                 !innerValue.IsNull() && innerValue.AsBoolean();
```

Comment:
I recommend extracting the logic here for getting the value for 'expanded' as a boolean variable, then setting expanded state to that value and collapsed to the opposite.